### ISSUE TITLE
Windows Output Path not working in v5.0.0

### ISSUE THREAD

**Issue by itstueben** [biesbjerg/ngx-translate-extract#171]
Hello,
in version 5 the output path is not working anymore.
The directorys are ignored on my Windows 10 Machine in IntellJ IDEA 2019

My npm Task looks:
` "translations:extract": "ngx-translate-extract --input ./src --output ./src/translations/de-DE.json ./src/translations/en-US.json --format=namespaced-json --clean --sort --fi \"'  '\"  ",
    `

Ouput:
...,/srctranslationsde-DE.json

the translations in file are correct

**Comment by biesbjerg** [comment]
Are you on Windows?

MacOS works as expected using your command


`./bin/cli.js --input /Users/kim/apps/test/src --output ./src/translations/de-DE.json ./src/translations/en-US.json --format=namespaced-json --clean --sort`

```
Found 61 strings.

Saving:
- /Users/kim/apps/ngx-translate-extract/src/translations/de-DE.json
- /Users/kim/apps/ngx-translate-extract/src/translations/en-US.json

Done.
```

**Comment by itstueben** [comment]
Yes.
 I think the Change [here ](https://github.com/biesbjerg/ngx-translate-extract/compare/v4.2.0...v5.0.0#diff-c9ee77b8b3575c61b785ba838750243bR57-R60) breaks it

**Comment by biesbjerg** [comment]
Hi again!

I just released v5.0.1 which solves the issue :)
